gammapy/modeling/models/spatial.py

```diff
@@ -1325,8 +1327,36 @@ def __init__(
 
         self._interp_kwargs = interp_kwargs
         self.filename = filename
+        kwargs["frame"] = self.map.geom.frame
+        if not "lon_0" in kwargs:
+            kwargs["lon_0"] = self.map_center.data.lon
+        if not "lat_0" in kwargs:
+            kwargs["lat_0"] = self.map_center.data.lat
         super().__init__(**kwargs)
 
+    def __str__(self):
+        width = self.map.geom.width
+        data_min = np.nanmin(self.map.data)
+        data_max = np.nanmax(self.map.data)
+
+        prnt = (
+            f"{self.__class__.__name__} model summary:\n "
+            f"Reference position: {self.position} \n "
```

Comment:
I think this is confusing. The WCS reference position doe snot change and is not equivalent to `self.position`.

Reply:
@adonath The reference was a bad naming. ( I was following the `LightCurveTemplateTemporalModel`, where it makes more sense). Updated to `Model position`